ApollosProject/apollos-apps
# fix: Update prayer algorithm

## DESCRIPTION
When there are no prayers requested within 24 hours and all prayers have been prayed for, no prayers would show in the app. 

### What does this PR do, or why is it needed?
This PR updates the prayer algorithm by replacing a hard coded data source method and exposing it as an argument.

This gives churches the flexibility to change the number of days since a prayer has been made to show up in the app. The default is now three days, instead of one.

### What design trade-offs/decisions were made?

### How do I test this PR?
On the `dailyPrayerAlgorithm` in `apollos-data-connector-rock/src/action-algorithms` console log the prayer request attributes in your terminal and confirm that the `EnteredDateTime gt datetime` = three days ago. 

For example, today is 2020-11-18 so I need to confirm that `EnteredDateTime gt datetime` = 2020-11-15.
![Screen Shot 2020-11-18 at 1 58 53 PM](https://user-images.githubusercontent.com/52924611/99575127-3545aa00-29a6-11eb-9681-a1f7cebfbb9a.png)

### What automated tests did you write?

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
